apps/client/src/common/models/AutomationSettings.ts

```diff
@@ -0,0 +1,9 @@
+import { AutomationSettings } from 'ontime-types';
+
+export const automationPlaceholderSettings: AutomationSettings = {
+  enabledAutomations: false,
+  enabledOscIn: false,
```

Comment:
@alex-Arc i am wondering if the OSC settings should be in the app settings instead of here

What do you think?

Reply:
yes 
at least it should be separate from the automations